PayPal/src/main/java/com/braintreepayments/api/PayPalInternalClient.java

```diff
@@ -50,9 +50,15 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
 
                     String requestBody = payPalRequest.createRequestBody(configuration, braintreeClient.getAuthorization(), successUrl, cancelUrl);
 
-                    braintreeClient.sendPOST(url, requestBody, new HttpResponseCallback() {
+                    braintreeClient.sendPOST(url, requestBody, new HTTPResponseCallback() {
+
                         @Override
-                        public void success(String responseBody) {
+                        public void onResult(String responseBody, Exception httpError) {
+                            if (httpError != null) {
```

Comment:
Nitpick - seems like all the other HttpResponseCallback null checks in this PR use `if (responseBody != null) { } else { } ` maybe we should follow the same pattern in this class?

Reply:
👍  most def will change.